src/content/en/updates/posts/2015/12/getting-started-pwa.markdown

```diff
@@ -18,19 +18,67 @@ There's been much welcome discussion about [Progressive Web Apps](https://infreq
 
 ## What is a Progressive Web App?
 
-> A Progressive Web App provides a native app-like user experience that is low friction and is built using modern web capabilities. Progressive Web Apps are hosted on the web and can progressively become an app on the user's device over time.
+> A Progressive Web App uses modern web capabilities to deliver an app-like user experience. They evolve from pages in browser tabs to immersive, top-level apps, maintaining the web's low friction at every moment.
 
```

Comment:
The first sentence is singular. The second is plural. They should agree in number.